src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -14,7 +14,7 @@ import org.joda.time.DateTime
 class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers {
   import driver.api._
 
-  ignore should "*REENABLE WITH MYSQL UNIT TESTS* crud entities" in withEmptyTestDatabase {
```

Comment:
the delete from a join raw sql that caused all of these unit tests to break is no longer needed